### ISSUE TITLE
improve nuvla documentation

### ISSUE THREAD

**Issue by cjdcordeiro** [nuvla/documentation#23]
- [ ] Nuvla Production Installation is still too long, cumbersome and hard to follow
 - [ ] Move all "Hands-on Exercises" into a 1st level section called "Tutorials"
 - [ ] CaaS Infrastructures are also too hard to follow, and introduce too many concepts that we shouldn't be explaining to a user. Links and references to external docs are preferred. This section must be simplified
 - [ ] "Manage Data" Demonstration for Emma...not sure if it fits there. Maybe move to the "Tutorials" sections, as "Demonstrations"
 - [ ] same for the "Running Applications with Data" Alice demonstration
 - [ ] for Felix, add the procedures for contributing to Nuvla*

**Comment by cjdcordeiro** [comment]
I'm not sure how to re-write this one: http://docs.nuvla.io/nuvla/customize-apps

**Comment by cjdcordeiro** [comment]
Tutorials section: https://github.com/nuvla/documentation/pull/24

**Comment by cjdcordeiro** [comment]
shall we also add `support@sixsq.com` somewhere ? Maybe on the bottom of the navbar, above the copyright...

**Comment by cjdcordeiro** [comment]
@0xbase12 @konstan @mebster I'm taking myself out as I don't have any other bullets to complete. Please keep updating this

**Comment by cjdcordeiro** [comment]
@0xbase12 @mebster please double check whether the missing boxes are still unticked

**Comment by mebster** [comment]
I've fixed the one for Felix.

**Comment by mebster** [comment]
@konstan could you look at the CaaS one?  With the latest machine feature, should this not be updated/re-written?

**Comment by mebster** [comment]
For the `customise-app`, I'm not sure. @cjdcordeiro @konstan any bright idea?
